.gitlab/benchmarks.yml

```diff
@@ -119,6 +119,11 @@ benchmark-http-propagation-inject:
   variables:
     SCENARIO: "http_propagation_inject"
 
+benchmark-rate-limiter:
+  extends: .benchmarks
+  variables:
+    SCENARIO: "rate_limiter"
```

Comment:
this is failing, saying:

> cannot stat '/app/baseline/benchmarks/rate_limiter/.': No such file or directory

probably because it is looking for the scenario on `main` instead of on this branch? 🤔

Reply:
that makes sense because benchmarks compare against the base branch